app/views/layouts/main.html.erb

```diff
@@ -52,7 +52,7 @@
 </div>
 <!-- wrap END -->
 
-<%= include_gon %>
+<%= include_gon(init: true) %>
```

Comment:
Nice! Didn't know about this option. Much better than checking everywhere if gon is initialized